a3/terraform/modules/cluster/gke/main.tf

```diff
@@ -63,11 +63,11 @@ module "dashboard" {
 module "resource_policy" {
   source = "../../common/resource_policy"
   for_each = {
-    for idx, node_pool in var.node_pools : "np-${idx}" => node_pool
+    for idx, node_pool in var.node_pools : idx => node_pool
```

Comment:
definitely a nitpick: `for_each` makes `module.resource_policy` a map that is addressed by strings instead of integers, and it is quite weird to see the addresses be the strings `["0", "2", "3"]` instead of integers. idk i just feel like it should either be integers or strings -- not integers as strings. obviously nbd if it works

Reply:
addressed.